modules/endpoint/README.md

```diff
@@ -0,0 +1,44 @@
+# IBM Container Registry endpoint
+
+This submodule allows you to retrieve the IBM Container Registry endpoint for a specific region.
```

Comment:
I would reference https://cloud.ibm.com/docs/Registry?topic=Registry-registry_overview#registry_regions in here too